manifests/cluster-authentication-operator_02_config.crd.yaml

```diff
@@ -5,10 +5,68 @@ metadata:
 spec:
   scope: Cluster
   group: operator.openshift.io
-  version: v1
   names:
     kind: Authentication
     plural: authentications
     singular: authentication
-  subresources:
-    status: {}
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  validation:
+    # Ensure we will be able to deserialize the object into the golang type
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          required: ["managementState"]
+          properties:
+            managementState:
+              type: string
+              enum:
```

Comment:
We need to tolerate "unset" (i.e. use the default) for every field.

Reply:
looking, ok

Reply:
uh, nevermind, looks like setting `default` is not allowed: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#validation

Reply:
it's safe to set "managementState" required, right? It will default to "Managed" based on `defaultOperatorConfig` [here](https://github.com/openshift/cluster-authentication-operator/blob/master/pkg/operator2/starter.go#L37).

Reply:
> it's safe to set "managementState" required, right? It will default to "Managed" based on `defaultOperatorConfig` [here](https://github.com/openshift/cluster-authentication-operator/blob/master/pkg/operator2/starter.go#L37).

No.  Any field can be left unset to indicate "give me the default."  Empty `spec` is valid:

```yaml
apiVersion: operator.openshift.io/v1
kind: Authentication
metadata:
  name: cluster
```

The existing code needs to be updated to handle this (i.e. it needs to do defaulting on inputs).

Reply:
ok